testing/zerotier-one/zerotier-one.initd

```diff
@@ -0,0 +1,20 @@
+#!/sbin/openrc-run
+
+ZEROTIERONE_PIDFILE="/var/run/${SVCNAME}.pid"
```

Comment:
Please read [man 8 openrc-run](http://manpages.org/openrc-run/8) and use declarative style instead of defining custom start/stop functions.

```sh
command="/usr/sbin/zerotier-one"
command_background="yes"
pidfile="/run/$RC_SVCNAME.pid"
```